tensorflow/lite/delegates/nnapi/acceleration_test_list.cc

```diff
@@ -60,7 +59,7 @@ FloatActivationsOpTest/Elu,30
 FloatActivationsOpTest/HardSwish
 QuantizedActivationsOpTest/HardSwish
 QuantizedActivationsOpTest/HardSwishBias
-QuantizedActivationsOpTest/Relu*
+QuantizedActivationsOpTest/Relu.+nt8
```

Comment:
Why are the int16 quatization test excluded ?

Reply:
From what I understand 16-bit tests are not supported by the tested accelerator (see other int16 exclusions in [acceleration_test_list.cc](https://github.com/tensorflow/tensorflow/blob/3581a0db862931065353e4285a1c6810ea70743e/tensorflow/lite/delegates/nnapi/acceleration_test_list.cc#L284) or https://github.com/tensorflow/tensorflow/pull/35997#issuecomment-611223883).